_automating-configurations/api/get-workflow-steps.md

```diff
@@ -30,18 +30,21 @@ OpenSearch validates workflows by using the validation template that lists the r
 }
 ```
 
-The Get Workflow Steps API retrieves this file.   
+The Get Workflow Steps API retrieves [this](https://github.com/opensearch-project/flow-framework/blob/2.x/src/main/java/org/opensearch/flowframework/workflow/WorkflowStepFactory.java#L120-L229) enum.   
```

Comment:
The line numbers on this link may change in the future.  It's also not very explanatory.  Should be something like, "Returns a list of Workflow Steps including their required inputs, outputs, default timeout value, ...." etc.

Reply:
Changed the description. Also, added link of `2.13` release branch to avoid changes later.